src/components/drawing/index.js

```diff
@@ -598,36 +597,32 @@ drawing.bBox = function(node) {
         return Lib.extendFlat({}, savedBBoxes[saveNum.value]);
     }
 
-    if(!drawing.test3) {
-        drawing.test3 = d3.select('#js-plotly-tester');
-        drawing.tester = drawing.test3.node();
-    }
+    var test3 = d3.select('#js-plotly-tester'),
+        tester = test3.node();
 
     // copy the node to test into the tester
     var testNode = node.cloneNode(true);
-    drawing.tester.appendChild(testNode);
+    tester.appendChild(testNode);
     // standardize its position... do we really want to do this?
     d3.select(testNode).attr({
         x: 0,
         y: 0,
         transform: ''
     });
 
-    var testRect = testNode.getBoundingClientRect();
-    if(!drawing.refRect) {
-        drawing.refRect = drawing.test3.select('.js-reference-point')
+    var testRect = testNode.getBoundingClientRect(),
+        refRect = test3.select('.js-reference-point')
             .node().getBoundingClientRect();
```

Comment:
If you want to keep the perf gains from #1585 in this fix, I'm pretty sure the only change we'd need is to call this `.getBoundingClientRect()` with each `drawing.bBox` call, rather than caching it. But we can still cache `tester`, and we can cache `test3.select('.js-reference-point').node()`.

Just a question of timing really - now that you have a scrolling test in place it seems safe to me, but you can leave it as is for now if you need to move on to other things.